app/controllers/orders_controller.rb

```diff
@@ -82,4 +83,15 @@ def update_status
   rescue StandardError => e
     handle_exception e
   end
+
+  def search_orders
+    @q = Product.by_product_of_order(current_user.id)
+                .ransack(params[:q])
+    @pro_ids = Array.new
+    @q.result.each do |product|
+      @pro_ids << product.id
+    end
```

Comment:
```suggestion
    @q.result.pluck(:product_id)
```
Như này được ko e ơi

Reply:
@anlvq-1058 cái này hay quá anh :p